src/App.js

```diff
@@ -0,0 +1,121 @@
+import React, { Component } from 'react';
+import './App.css';
+import Clock from "./components/Clock"
+import Navbar from "./components/Navbar"
+import ActionRow from "./components/ActionRow"
+import AdjustRow from "./components/AdjustRow"
+import MusicRow from './components/MusicRow'
+
+export class App extends Component {
+  constructor(){
+    super()
+    this.state={
+      workTime:{
+        minutes:'25',
+        seconds:'00'
+      },
+      breakTime:{
+        minutes:'05',
+        seconds:'00'
+      },
+      minutes:'25',
+      seconds:'00', 
+      timer: null, 
+      isWorkMode:true, 
+      isActive:false
+    }
+  }
+
+  start = () =>{
+    this.setState({
+      isActive:true,
+      timer:setInterval(()=>{
+        this.tick()
+        },1000)
+      })
+  }
+
+  pause = () =>{
+    clearInterval(this.state.timer)
+    this.setState({isActive:false})
+  }
+
+  reset = () =>{
+    let minutes = this.state.isWorkMode ? this.state.workTime.minutes : this.state.breakTime.minutes
+    let seconds = this.state.isWorkMode ? this.state.workTime.seconds : this.state.breakTime.seconds
+    this.setState({isActive:false, minutes, seconds})
+    clearInterval(this.state.timer)
+  }
+
+  modifyTime = (seconds,minutes) =>{
+    seconds = parseInt(seconds)
+    minutes = parseInt(minutes)
+
+    if(seconds > 0){
+      seconds--
+    } else {
+      minutes--
+      seconds = 59
+    }
+
+    seconds = seconds < 10 ? `0${seconds}` : seconds
+    minutes = minutes < 10 ? `0${minutes}` : minutes
+    return {seconds,minutes}
+  }
+
+  tick = () =>{
+    let {seconds , minutes} = this.modifyTime(this.state.seconds ,this.state.minutes)
+    this.setState({seconds , minutes})
+    if(minutes === '00' && seconds === '00'){
+      this.finished()
+    }
+  }
+
+  
+  changeMode = mode =>{
+    let isWorkMode = (mode === 'work') ? true : false
```

Comment:
can be shorter: `let isWorkMode = (mode === 'work')` becouse (mode === 'work') returns true or false